components/Cast/Cast.tsx

```diff
@@ -12,7 +12,7 @@ const Cast = ({ cast = {} as CastType }: { cast: CastType }) => {
     <div className="flex flex-col gap-3 mb-[20px] border border-500-[#ddd] p-[10px]">
       <AuthorDetails pfpUrl={cast.author.pfp_url} displayName={cast.author.display_name} />
       <div className="flex items-center w-full">
-        <Upvote target={hash as Address} />
+        <Upvote target={hash as Address} cast={cast} />
```

Comment:
do we need to pass `hash` if we are also passing `cast`?

cast already contains `hash` right?